mlrun/feature_store/api.py

```diff
@@ -261,12 +265,36 @@ def ingest(
             raise mlrun.errors.MLRunInvalidArgumentError(
                 "data source was not specified"
             )
+
+        if source.schedule:
+            featureset.reload(update_spec=False)
+            if featureset.status.targets and featureset.status.targets[0].last_written:
+
+                min_time = datetime.fromisoformat(
+                    featureset.status.targets[0].last_written
+                )
+                for target in featureset.status.targets:
+                    if target.last_written:
+                        cur_last_written = datetime.fromisoformat(target.last_written)
```

Comment:
i don't have a specific comment but something is worrying me about time zones conversion here (and in datetime.now)
need to think it through